cfme/tests/cloud_infra_common/test_retirement.py

```diff
@@ -157,11 +194,46 @@ def test_retirement_now(retire_vm):
     retire_times = dict()
     retire_times['start'] = generate_retirement_date_now() + timedelta(minutes=-5)
     retire_vm.retire()
+
+    # Verify flash message
+    view = retire_vm.appliance.browser.create_view(RequestsView)
+    assert view.is_displayed
+    view.flash.assert_success_message(
+        "Retirement initiated for 1 VM and Instance from the CFME Database")
+
     verify_retirement_state(retire_vm)
     retire_times['end'] = generate_retirement_date_now() + timedelta(minutes=5)
     verify_retirement_date(retire_vm, expected_date=retire_times)
 
 
+@pytest.mark.rhv1
+def test_retirement_now_multiple(retire_vm_pair, provider):
+    """Tests on-demand retirement of two instances/vms from All VMs page
+
+    Polarion:
+        assignee: tpapaioa
+        casecomponent: Provisioning
+        initialEstimate: 1/6h
+    """
+    retire_times = dict()
+    retire_times['start'] = generate_retirement_date_now() + timedelta(minutes=-5)
+    retire(retire_vm_pair)
+
+    # Verify flash message
+    view = retire_vm_pair[0].appliance.browser.create_view(RequestsView)
```

Comment:
You can simplify this chain - and call create_view straight from retire_vm_pair or from appliance.browser.

Reply:
retire_vm_pair is a list, but I've defined 'collection' as a reference to the vm collection retire_vm_pair[0].parent, which is used for calling create_view as well as the collection-level retire and set_retirement_date methods (see comment below).